test/podFailover/workload/cmd/main.go

```diff
@@ -67,15 +68,18 @@ func main() {
 	flag.Parse()
 	var config *rest.Config
 	var err error
```

Comment:
It doesn't seem like we need to declare this variable outside the scope of the `if` statement.

Reply:
the scope of config creations is weird within different flows so the defined config at the top makes sure all versions of config are instantiated to the same variable

Reply:
I was referring to the `err` variable. This is non-blocking.